modules/kernel/src/main/scala/trace4cats/kernel/headers/GoogleCloudTraceToHeaders.scala

```diff
@@ -47,7 +50,7 @@ private[trace4cats] object GoogleCloudTraceToHeaders {
         traceId = traceId,
         spanId = spanId,
         parent = none,
-        traceFlags = TraceFlags(if (enabled == "1") SampleDecision.Include else SampleDecision.Drop),
+        traceFlags = TraceFlags(if (enabled == Some("1")) SampleDecision.Include else SampleDecision.Drop),
```

Comment:
Here "0" means drop - absence means use `SpanSampler` to make decision.

Reply:
How is it working in the original code then? I dont see it being set to None

Reply:
@sideeffffect do we really want to change the old behaviour where it is always SampleDecision.Drop except for explicitly set "1"?

Reply:
The old behavior allowed only for `1` and `0`.

Me and @bambuchaAdm agreed that it's sensible to drop only if explicitly requested (using `0`). Otherwise keep the trace.